src/screens/journey.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { useEffect } from 'react';
+import { StyleSheet, View, } from 'react-native';
+import { Text, Button } from 'react-native-paper';
+import { config } from '../config';
+import { ScreenNavigationProps } from '../routes';
+import TicketFlatList from '../components/ticketsFlatList';
+import { JourneyOutput } from '../types/JourneyOutput';
+
+const baseUrl = 'https://mobile-api-softwire2.lner.co.uk';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#eee',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  text: {
+    paddingBottom: 24,
+  },
+});
+
+type JourneyScreenProps = ScreenNavigationProps<'Journey'>;
+
+type JourneyResponse = {
+  numberOfAdults: number;
+  nextOutboundQuery: string;
+  outboundJourneys: Array<OutboundJourney>;
+};
+
+type OutboundJourney = {
+  journeyId: string;
+  departureTime: string;
+  arrivalTime: string;
+  primaryTrainOperator: TrainOperator;
+  tickets: Array<Tickets>;
+};
+
+type TrainOperator = {
+  code: string;
+  name: string;
+};
+
+type Tickets = {
+  name: string;
+  priceInPennies: number;
+};
+
+const defaultJourneyOutput: JourneyOutput = {
```

Comment:
I think I'd prefer to make the default journey output `null`, and then handle that nullness 'last minute' in the UI code. A `JourneyOutput` with these values feels like a surprise and may cause issues somewhere else if it's passed around and assumed to be a normal journey